client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -20,9 +21,11 @@ function Dashboard() {
       messages={<BrowserChecker />}
       header="Dashboard"
       headerActions={
-        <Link to={`${ROUTES.WORKBENCH}${ROUTES.ACCOUNT}`} className="wb-link">
-          Manage Account
-        </Link>
+        (!hideManageAccount &&
+          <Link to={`${ROUTES.WORKBENCH}${ROUTES.ACCOUNT}`} className="wb-link">
+            Manage Account
+          </Link>
+        )
       }
       contentClassName={styles['panels']}
       contentLayoutName="twoColumn"
```

Comment:
```suggestion
      contentLayoutName="twoColumnUnequal"
```

**But**, this would bring back a bug we hid under the rug by going to 50/50 ([FP-1572](https://jira.tacc.utexas.edu/browse/FP-1572) & [#626](https://github.com/TACC/Core-Portal/pull/626)).

I'll look into it.

Reply:
I have brought back unequal layout code (with new fix) in https://github.com/TACC/Core-Portal/pull/689.